`ENOENT` of `eslint` causes a silent exit rather than an expected error message to install `eslint`

### Environment

  - Node Version: 16.13.0 & 14.17.1
  - Eslint-Watch Version: 7.0.0 & 8.0.0
  - Eslint Version:

### Basic Description of the problem

When `eslint` is missing, `eslint-watch` exits with no message.

The solution is to re-install `eslint`, as hinted at by an error message located in the source. However, this error message is not being emitted and instead the program silently exits.

It looks like the `error.errno` (:one:) is `-2`, where the `error.code` is `ENOENT`. The `error.stdout` and `error.stderr` (:two:) in this case are either `null` or `''`, meaning when the `err.message` is emitted (3️⃣ ), it's blank.

:one:
https://github.com/rizowski/eslint-watch/blob/59d480912c2cb972d83cab10e7c27bd1a5330e74/src/eslint/index.js#L27-L29

:two:
https://github.com/rizowski/eslint-watch/blob/59d480912c2cb972d83cab10e7c27bd1a5330e74/src/eslint/index.js#L31

3️⃣ 
https://github.com/rizowski/eslint-watch/blob/59d480912c2cb972d83cab10e7c27bd1a5330e74/bin/esw#L14

### How to reproduce it

  1. Delete `node_modules/.bin/eslint`
  2. Call `esw`

### Debug output:

```
> <my-app>@0.1.0 lint /Users/<me>/development/projects/<my-app>
> <my-cli> lint 'src/**/*.{ts,tsx}'

  esw:eslint-help Loaded +0ms
  esw:main [
  esw:main   '--config',
  esw:main   '/Users/<me>/development/projects/<my-app>/.eslintrc.js',
  esw:main   'src/**/*.{ts,tsx}'
  esw:main ] +0ms
  esw:main ESW: v7.0.0 +1ms
  esw:eslint Executing [ '--help' ] +0ms
  esw:eslint Error: Command failed with ENOENT: eslint --help
  esw:eslint spawn eslint ENOENT
  esw:eslint     at Process.ChildProcess._handle.onexit (internal/child_process.js:269:19)
  esw:eslint     at onErrorNT (internal/child_process.js:467:16)
  esw:eslint     at processTicksAndRejections (internal/process/task_queues.js:82:21) +28ms

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! <my-app>@0.1.0 lint: `<my-cli> lint 'src/**/*.{ts,tsx}'`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the <my-app>@0.1.0 lint script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/<me>/.npm/_logs/2022-06-09T18_37_48_417Z-debug.log
```

--- a/src/eslint/index.js
+++ b/src/eslint/index.js
@@ -24,7 +24,7 @@ const eslint = {
     } catch (error) {
       logger.debug(error);
 
-      if (error.errno === 'ENOENT' && error.path === 'eslint') {
+      if (error.code === 'ENOENT' && error.path === 'eslint') {
         throw new Error("Error: Eslint was not found either globally or locally.\nRun 'npm i -g eslint' or 'npm i -D eslint' to resolve the issue.");
       }
 

